go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	github.com/djherbis/buffer v1.1.0
 	github.com/djherbis/nio/v3 v3.0.1
 	github.com/fatih/color v1.13.0
+	github.com/go-git/go-git/v5 v5.4.2
```

Comment:
I would use the latest version https://github.com/go-git/go-git/releases/tag/v5.9.0

Reply:
Unfortunately version after `v5.4.2` is causing some errors